QRScanner/QRScannerView.swift

```diff
@@ -268,7 +268,8 @@ extension QRScannerView: AVCaptureMetadataOutputObjectsDelegate {
             videoDataOutputEnable = true
 
             DispatchQueue.main.async { [weak self] in
-                self?.moveImageViews(qrCode: stringValue, corners: readableObject.corners)
+                self?.moveImageViews(corners: readableObject.corners)
+                self?.delegate?.success(stringValue)
```

Comment:
Add this line.
But if this should be called after image moving, this is called in `UIView.animate` completion block 
https://github.com/mercari/QRScanner/blob/9f5587d5e2bc530bee82f7c4e9d5c4dd6af333fd/QRScanner/QRScannerView.swift#L231

Reply:
^ is better if it is the same as iOS13+ design... I think